src/pyhf/readxml.py

```diff
@@ -23,13 +23,14 @@ def extract_error(h):
     bin uncertainties are then Poisson, and so the `sqrt(entries)`.
 
     Args:
-        h (uproot.rootio.TH1 object): The histogram
+        hist (`<dynamic>.Model_TH1F_v2`): The histogram
```

Comment:
That dynamic class will be different for each ROOT file you open. However, all 1-dimensional histograms will be subclasses of `uproot.behaviors.TH1.TH1`.

Reply:
Thanks. I'll change this.